src/lib/charm/openstack/keystone_saml_mellon.py

```diff
@@ -292,19 +295,20 @@ def configuration_complete(self):
 
         return all(required_config.values())
 
-    def assess_status(self):
-        """Determine the current application status for the charm"""
-        hookenv.application_version_set(self.application_version)
+    def custom_assess_status_check(self):
+        """Custom asses status.
+
+        Check the configuration is complete.
+        """
         if not self.configuration_complete():
             errors = [
                 '{}: {}'.format(k, v)
                 for k, v in self.options.validation_errors.items() if v]
```

Comment:
AFAICT, the `validation_errors` property on the config adapter does the check for `v`?

Reply:
Done.